### PR TITLE
Added Quick Start section

### PR DESCRIPTION
**Pull request by dbooth-boston**
Added a simple Quick Start section to the README.md (via .verb.md).

### CODE REVIEW

**Comment by dbooth-boston** [comment]
This addresses issues #98 and #99.

File: .verb.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## Quick Start 
+Assuming you want to add a TOC to README.md:
+1. ```$ npm install -g markdown-toc```
+2. Edit README.md and insert the following line where you want the TOC inserted:<br />```<-- toc -->```
```
Comment (glynhudson) [inline .verb.md]: This should be 

```<!-- toc -->```

Missing `!`

Comment (dbooth-boston) [reply .verb.md]: Oops, good catch!   I'll fix the PR in a moment ...

Comment (glynhudson) [reply .verb.md]: No problem. Your quick start was very useful to me as a new user 👍

File: .verb.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## Quick Start 
+Assuming you want to add a TOC to README.md:
+1. ```$ npm install -g markdown-toc```
```
Comment (doowb) [inline .verb.md]: You only need 1 backtick when the code is inline.

File: .verb.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
...
+2. Edit README.md and insert the following line where you want the TOC inserted:<br />```<!-- toc -->```
+3. ```$ markdown-toc -i README.md```
```
Comment (doowb) [inline .verb.md]: You only need 1 backtick when the code is inline.

**Comment by doowb** [comment]
Thanks @dbooth-boston! I requested some changes in a review. Once those are done, I'll merge.

**Comment by dbooth-boston** [comment]
Ok @doowb it should be fixed now.  I corrected the backticks and reverted the README.md .  Thanks!

**Comment by dbooth-boston** [comment]
@doowb it looks like this is approved and ready to merge now.

**Comment by doowb** [comment]
@dbooth-boston thanks for the ping. I'll merge and get the readme updated soon.

**Comment by TT--** [comment]
https://github.com/jonschlinkert/markdown-toc/pull/96

**Pull request merged**